.github/workflows/CI-windows-msys2.yml

```diff
@@ -51,9 +51,6 @@ jobs:
           cache: false
           msystem: ${{ inputs.msys2-msystem }}
           update: false
```

Comment:
As per https://github.com/msys2/setup-msys2?tab=readme-ov-file#update,
that means we don't update the package index either? That seems unfortunate.